src/Uno.UI/DataBinding/BindingPropertyHelper.FastConvert.cs

```diff
@@ -996,10 +996,14 @@ private static bool FastStringToTypeConvert(Type outputType, string input, ref o
 			}
 		}
 
-		private static readonly Regex _splitRegex = new Regex(@"\s*,\s*|\s+");
+		private static Regex _splitRegex = null;
 
 		static string[] Split(string value)
-			=> _splitRegex.Split(value);
+		{
+			_splitRegex ??= new Regex(@"\s*,\s*|\s+");
```

Comment:
I'm curious, what's the perf boost here? Just not creating the Regex if it's not needed, or something more subtle? Might also warrant a comment so it's not refactored away by some tidy mind in the future.

Reply:
That's fair. In practice I'd prefer removing the regex altogether because it's quite costly for the type of split operation we're making. (I'll keep #6877 opened)